packages/framework-provider-kubernetes/src/services/event-registry.ts

```diff
@@ -2,11 +2,14 @@
 import { EventEnvelope, Logger } from 'framework-types/dist'
 import { v4 as uuidv4 } from 'uuid';
 import fetch from 'node-fetch'
+//import { RedisAdapter } from './redis-adapter'
 
 export class EventRegistry {
   constructor(readonly url: string) {}
 
   public async store(event: EventEnvelope, logger: Logger): Promise<void> {
+    //await RedisAdapter.build().keys('*', logger)
+
```

Comment:
Left some commented code

Reply:
I left it intended to know how to use it. I'll remove it on next PR